docs/book/src/reference/versions.md

```diff
@@ -63,19 +70,19 @@ These diagrams show the relationships between components in a Cluster API releas
 
 #### Core Provider (`cluster-api-controller`)
 
-|                   | v0.3 (v1alpha3)      | v0.4 (v1alpha4)   | v1.0 (v1beta1)    | v1.1 (v1beta1)    | v1.2 (v1beta1)    | v1.3 (v1beta1)    | v1.4 (v1beta1)    |
-|-------------------|----------------------|-------------------|-------------------|-------------------|-------------------|-------------------|-------------------|
-| Kubernetes v1.16  | ✓                    |                   |                   |                   |                   |                   |                   |
-| Kubernetes v1.17  | ✓                    |                   |                   |                   |                   |                   |                   |
-| Kubernetes v1.18  | ✓                    | ✓ (only workload) | ✓ (only workload) | ✓ (only workload) | ✓ (only workload) | ✓ (only workload) | ✓ (only workload) |
-| Kubernetes v1.19  | ✓                    | ✓                 | ✓                 | ✓                 | ✓ (only workload) | ✓ (only workload) | ✓ (only workload) |
-| Kubernetes v1.20  | ✓                    | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 |
-| Kubernetes v1.21  | ✓                    | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 |
-| Kubernetes v1.22  | ✓ (only workload)    | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 |
-| Kubernetes v1.23* |                      | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 | ✓                 |
-| Kubernetes v1.24  |                      |                   |                   | ✓                 | ✓                 | ✓                 | ✓                 |
-| Kubernetes v1.25  |                      |                   |                   |                   | ✓                 | ✓                 | ✓                 |
-| Kubernetes v1.26  |                      |                   |                   |                   | ✓                 | ✓                 | ✓                 |
+|                   | v0.3 (v1alpha3)   | v0.4 (v1alpha4)   | v1.0 (v1beta1)    | v1.1 (v1beta1)    | v1.2 (v1beta1)    | v1.3 (v1beta1)    | v1.4 (v1beta1)    |
```

Comment:
Q: should we add an out of support notice below minors?
Also, at some point, we should also consider dropping columns from those tables

Reply:
I added (EOL) as (out of support) requires another two lines in the table header

Reply:
+1 to drop the columns, if someone want to look at older version support, they can look at holder versions of the book

Reply:
Sounds good, follow-up: https://github.com/kubernetes-sigs/cluster-api/pull/8168